input/examples/medicationrequest-simvastatin.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<MedicationRequest xmlns="http://hl7.org/fhir">
+    <id value="simvastatin"/>
+    <meta>
+        <profile value="http://hl7.org.au/fhir/core/StructureDefinition/au-core-medicationrequest"/>
+    </meta>
+    <status value="active"/>
+    <intent value="order"/>
+    <medicationReference>
+        <reference value="Medication/simvastatin"/>
```

Comment:
Medication-simavastin is another non-AU core example that needs to be made AU Core. Clean it up, remove RX norm etc

Reply:
Done